charts/osschallenge/ci/default-values.yaml

```diff
@@ -5,3 +5,6 @@ postgresql:
 osschallenge:
   settings:
     emailPassword: s3cr3t
+persistence:
+  storageClassName: "default"
+  enabled: false
```

Comment:
This should disable it, right?

Reply:
You need to implement the flag in the `pvc.yaml` template as well:
```
{{- if .Values.persistence.enabled }}
apiVersion: v1
# ...
{{- end }}
```

Reply:
ah ok, I see

Reply:
You need to wrap the actual volume stuff in the deployment in a similar `if` block. I think the postgresql chart has an example where an empty volume is used if persistence is disabled.